390/Serena_Li.Rmd

```diff
@@ -0,0 +1,40 @@
+---
+title: "MP2 Viz"
+author: "Serena Li"
+date: "9/15/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r}
+library(tidyverse)
+library(tsibble)
+library(lubridate)
+library(fpp3)
+library(dplyr)
+library(readr)
+```
+
+```{r}
+stem <- read_csv("tree_main_census/data/census-csv-files/scbi.stem1.csv")
+```
+
+```{r}
+avg_dbh<-stem %>% 
+  filter(!is.na(dbh),!is.na(sp)) %>% 
+  mutate(dbh_num=as.numeric(as.character(dbh))) %>% 
+  group_by(sp) %>% 
+  mutate(count=n(),
```

Comment:
Remove `group_by` and `mutate()` here and boxplot the raw values

Reply:
Note here that once you're done with a particular comment, you can click the "Resolve conversation" button to remove this from this discussion. It makes keeping track of what's left to do cleaner.